tasks/copy_files_to_archive/test/test_copy_files_to_archive_postgres.py

```diff
@@ -59,6 +61,7 @@ def tearDown(self):
         requests_db.get_utc_now_iso = self.mock_utcnow
         boto3.client = self.mock_boto3_client
         try:
+            os.environ.pop('PREFIX')
             del os.environ['COPY_RETRY_SLEEP_SECS']
```

Comment:
Not necessary to change here, but would it make sense to change all the `del` commands to `os.environ.pop` at some point?

Reply:
I certainly would. I think avoiding arcane testing errors takes higher priority than checking if a task deleted an environment variable for some reason.